  2024-11-30T06:53:32.954842Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-11-30T06:53:32.954923Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-11-30T06:53:32.954927Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-11-30T06:53:32.956630Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-11-30T06:53:32.956637Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-11-30T06:53:32.958112Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T10:34:26.044795Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T10:34:26.044888Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T10:34:26.044890Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T10:34:26.046667Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T10:34:26.046674Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T10:34:26.048175Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T10:34:39.842694Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T10:34:42.224906Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T10:34:42.795172Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T10:34:52.251776Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T10:43:51.413843Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T10:43:51.413886Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T10:43:51.413888Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T10:43:51.415434Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T10:43:51.415440Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T10:43:51.416667Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T10:43:59.969400Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T10:43:59.969446Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T10:43:59.969448Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T10:43:59.971162Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T10:43:59.971210Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T10:43:59.973739Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T10:47:56.050235Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T10:47:56.050279Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T10:47:56.050280Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T10:47:56.052172Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T10:47:56.052179Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T10:47:56.053563Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T10:48:46.394465Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T10:48:46.394513Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T10:48:46.394515Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T10:48:46.396241Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T10:48:46.396248Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T10:48:46.397616Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T10:55:01.905979Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T10:55:01.906000Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T10:55:01.906002Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T10:55:01.906763Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T10:55:01.906777Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T10:55:01.908887Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T10:59:47.988557Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T10:59:47.988600Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T10:59:47.988601Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T10:59:47.990385Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T10:59:47.990392Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T10:59:47.991589Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T11:05:48.089676Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:07:35.550966Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:09:09.283023Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:09:11.303180Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:09:15.235626Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:14:14.131862Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:14:23.535440Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:14:38.475674Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:16:08.131566Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:16:23.074472Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:16:33.173564Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:16:34.460499Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:16:36.926786Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:17:49.086865Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:17:53.268108Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:18:06.166939Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:18:56.256442Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T11:18:56.470300Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T11:19:01.644420Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:01.678529Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:01.986715Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:02.056889Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:02.124589Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:02.897876Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:03.918879Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:05.632253Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:06.154646Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:17.658513Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:17.794419Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:17.862119Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:18.213904Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:18.761646Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:18.900110Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:19.001587Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:19.069424Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:19.138166Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:19.446452Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:19.480098Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:41.460807Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:42.231871Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:50.614014Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:19:53.229190Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:10.592819Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:12.342152Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:13.054965Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:18.924270Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:28.706905Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:29.295058Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:29.953954Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:38.111173Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:38.943274Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:40.465870Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:41.016621Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:42.935411Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:43.912416Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:46.821041Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:49.013153Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:50.501766Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:50.560991Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:50.628379Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:52.449357Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:53.075404Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:53.370147Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:54.061179Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:54.529427Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:55.059831Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:57.011338Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:57.673033Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:58.793183Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:20:59.112925Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:02.104344Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:06.337301Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:17.674689Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:19.222734Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:23.051358Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:23.357433Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:24.416311Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:24.699735Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:25.195528Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:25.630206Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:36.995682Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:37.299690Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:39.259929Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:50.727862Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:50.994453Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:21:53.715475Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:22:09.788060Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:22:10.398483Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:22:10.501634Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:22:10.677890Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:24:59.619709Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:25:29.862614Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:25:45.855080Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T11:25:45.855114Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T11:25:45.855116Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T11:25:45.855974Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T11:25:45.855985Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T11:25:45.857486Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T11:32:09.950107Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T11:32:09.950122Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T11:34:53.276810Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:53.311908Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:53.725605Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:56.682790Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:56.716991Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:57.027039Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:57.093487Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:57.550082Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:58.043783Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:58.596459Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:58.699159Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:58.835707Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:34:59.149700Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:05.708306Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:05.771848Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:05.910705Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:06.012034Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:06.080084Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:06.148454Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:06.459546Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:06.492687Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:35:32.388106Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:35:54.684941Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:40:07.204392Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:40:07.239369Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:40:08.101961Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:40:08.135379Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T11:41:40.639255Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\NTUSER.DAT
C:\Users\LENOVO\ntuser.dat.LOG1
C:\Users\LENOVO\ntuser.dat.LOG2
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TM.blf
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000001.regtrans-ms
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000002.regtrans-ms
C:\Users\LENOVO\ntuser.ini
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:41:44.011663Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\NTUSER.DAT
C:\Users\LENOVO\ntuser.dat.LOG1
C:\Users\LENOVO\ntuser.dat.LOG2
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TM.blf
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000001.regtrans-ms
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000002.regtrans-ms
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T11:52:09.858457Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Program
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:00:44.941369Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:01:00.944853Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:01:02.936791Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
C:\DumpStack.log.tmp
C:\hiberfil.sys
C:\pagefile.sys
C:\swapfile.sys
C:\SYSTAG.BIN
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:01:03.510662Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:01:03.779362Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
C:\DumpStack.log.tmp
C:\hiberfil.sys
C:\pagefile.sys
C:\swapfile.sys
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:01:04.689498Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:01:15.616599Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\NTUSER.DAT
C:\Users\LENOVO\ntuser.dat.LOG1
C:\Users\LENOVO\ntuser.dat.LOG2
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TM.blf
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000001.regtrans-ms
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000002.regtrans-ms
C:\Users\LENOVO\ntuser.ini
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:01:16.287726Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\NTUSER.DAT
C:\Users\LENOVO\ntuser.dat.LOG1
C:\Users\LENOVO\ntuser.dat.LOG2
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TM.blf
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000001.regtrans-ms
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000002.regtrans-ms
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:01:18.156657Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\NTUSER.DAT
C:\Users\LENOVO\ntuser.dat.LOG1
C:\Users\LENOVO\ntuser.dat.LOG2
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TM.blf
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000001.regtrans-ms
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000002.regtrans-ms
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:02:07.759914Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\NTUSER.DAT
C:\Users\LENOVO\ntuser.dat.LOG1
C:\Users\LENOVO\ntuser.dat.LOG2
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TM.blf
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000001.regtrans-ms
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000002.regtrans-ms
C:\Users\LENOVO\ntuser.ini
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:05:04.357482Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:05:04.357506Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:05:08.767069Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:05:12.172102Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:05:12.274295Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:05:12.308534Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:05:12.412808Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:05:12.445114Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:05:30.162063Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:05:30.875056Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:05:43.303952Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\Searches\winrt--{S-1-5-21-862151880-4168968873-430351908-1001}-.searchconnector-ms
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:05:44.459298Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\Searches\winrt--{S-1-5-21-862151880-4168968873-430351908-1001}-.searchconnector-ms
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:05:45.163193Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\Searches\winrt--{S-1-5-21-862151880-4168968873-430351908-1001}-.searchconnector-ms
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:05:48.720806Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\Searches\winrt--{S-1-5-21-862151880-4168968873-430351908-1001}-.searchconnector-ms
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:07:31.226699Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:31.934091Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:31.968279Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:32.070959Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:32.106210Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:32.208662Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:32.348165Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:32.592381Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:33.121061Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:35.079790Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:07:36.375412Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:26.244344Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:28.899145Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:30.121101Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:30.491490Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:30.835885Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:32.731713Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:33.420163Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:34.670994Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:44.815288Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:08:44.815344Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:08:52.325467Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:08:57.119568Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:00.020513Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:01.247855Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:01.691408Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:03.254375Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:04.640342Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:05.710187Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:06.638601Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:07.587477Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:07.854247Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:08.004153Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:08.204178Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:08.384432Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:09:08.574323Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:05.983842Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:06.355701Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:07.148500Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:08.339983Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:15.109993Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:10:15.110291Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:10:27.036934Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:27.713063Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:28.585441Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:28.743932Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:28.912738Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:30.346578Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:30.496300Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:10:30.746689Z ERROR yazi_plugin::utils::log: "spawn `7z` and `7zz` both commands failed, error code: nil"
    at yazi-plugin\src\utils\log.rs:20

  2024-12-01T12:11:16.956899Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T12:11:16.956942Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T12:11:16.956944Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T12:11:16.958831Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T12:11:16.958839Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T12:11:16.960081Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T12:11:19.012149Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:11:19.012482Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:12:35.603283Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:12:35.603691Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T12:13:27.778425Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
C:\vfcompat.dll
C:\appverifUI.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:13:39.514702Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:13:43.774399Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:14:08.687671Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:18:19.282838Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T12:18:19.282902Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T12:18:19.282904Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T12:18:19.283652Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T12:18:19.283667Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T12:18:19.285637Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T12:28:31.126425Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T12:28:31.126483Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T12:28:31.126486Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T12:28:31.127582Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T12:28:31.127600Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T12:28:31.130625Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T12:33:10.303782Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T12:33:10.303802Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T12:33:10.303804Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T12:33:10.305403Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T12:33:10.305409Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T12:33:10.306808Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T12:33:13.197875Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:47:48.406690Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Program
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:49:36.639556Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\Files
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:49:37.247453Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\(x86)\clink
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:49:41.372101Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Program
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:49:42.104989Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\Files
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T12:49:42.790338Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\(x86)\clink
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T14:03:36.947566Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T14:03:36.947613Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T14:03:36.947616Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T14:03:36.948732Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T14:03:36.948742Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T14:03:36.950118Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T15:55:42.241125Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T15:55:44.199790Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T15:55:51.100742Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T15:56:38.978827Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T15:56:42.949073Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T15:56:48.161600Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\NTUSER.DAT
C:\Users\LENOVO\ntuser.dat.LOG1
C:\Users\LENOVO\ntuser.dat.LOG2
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TM.blf
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000001.regtrans-ms
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000002.regtrans-ms
C:\Users\LENOVO\ntuser.ini
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:01:20.154803Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:01:27.731236Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:01:27.757220Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:01:27.790666Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:01:27.824519Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:01:27.857706Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:01:27.891178Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:01:27.974791Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:53:15.993508Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:53:21.814878Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:53:23.769014Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T16:53:35.185791Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T16:53:35.185817Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T16:53:35.185819Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T16:53:35.186827Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T16:53:35.186838Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T16:53:35.188565Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T16:54:40.605950Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T16:54:40.605970Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T16:54:40.605971Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T16:54:40.607874Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T16:54:40.607881Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T16:54:40.609222Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T16:59:14.399357Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T16:59:14.399377Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T16:59:14.399378Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T16:59:14.401500Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T16:59:14.401519Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T16:59:14.403109Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T17:09:18.582988Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Program
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T17:09:18.597836Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T17:18:49.931969Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T17:18:49.932022Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T17:18:49.932024Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T17:18:49.933765Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T17:18:49.933788Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T17:18:49.935920Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T17:21:13.144476Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T17:21:13.144497Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T17:21:13.144498Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T17:21:13.145882Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T17:21:13.145898Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T17:21:13.148006Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T17:23:14.292232Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T17:23:14.292252Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T17:23:14.292254Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T17:23:14.293402Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T17:23:14.293410Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T17:23:14.294475Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T17:23:18.852264Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:21:31.975924Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T18:21:31.975971Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T18:21:31.975972Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T18:21:31.977913Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T18:21:31.977924Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T18:21:31.979339Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T18:29:06.161823Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T18:29:06.161875Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T18:29:06.161877Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T18:29:06.163278Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T18:29:06.163291Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T18:29:06.165658Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T18:32:37.774795Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:32:42.828765Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:32:48.076359Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:32:52.920368Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:33:39.295458Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:33:42.773529Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:33:45.014400Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:36:06.787689Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:39:41.982124Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T18:47:50.766731Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T18:47:50.766923Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T18:47:53.431306Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:47:53.465539Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:47:53.819054Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:47:57.341248Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:47:57.547515Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:47:57.882010Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:48:01.543692Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:48:25.769937Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-01T18:48:25.769980Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-01T18:48:25.769982Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-01T18:48:25.771896Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-01T18:48:25.771903Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-01T18:48:25.773213Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-01T18:48:26.880913Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T18:48:26.880976Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T18:48:28.694650Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:48:29.631743Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:48:32.112828Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:52:24.124759Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T18:52:24.124895Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T18:52:25.286187Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:52:25.292532Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:52:25.706623Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:52:26.609712Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:52:26.829693Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:52:30.598771Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:53:53.913737Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:59:01.655627Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:59:01.863931Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:59:03.811791Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:59:03.816639Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:59:03.953427Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T18:59:35.771939Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:00:50.053424Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:00:50.053465Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:00:51.358507Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:00:51.563933Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:00:51.915614Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:15: attempt to index a nil value (local 'child')\nstack traceback:\n\t[string \"ouch\"]:15: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:22.940494Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:01:22.940526Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:01:24.204073Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:26.980583Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:28.141199Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:28.907176Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:33.777074Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:36.529361Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:37.810683Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:39.360231Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:40.816100Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:43.197624Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:43.639764Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:45.125815Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:48.643298Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:49.300812Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:50.812589Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:51.179270Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:54.649872Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:56.201723Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:57.006856Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:57.797417Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:58.322561Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:58.620050Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:01:59.453372Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:03:29.754607Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:03:38.440133Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:03:40.911755Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:03:41.649182Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:03:48.400541Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:06:19.672568Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:06:19.672645Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:06:20.983983Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:06:23.475497Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:06:23.879270Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:06:29.894373Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:06:30.385444Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:06:30.553718Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:06:30.772965Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:06:31.550763Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:08:58.237742Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:08:58.403564Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:08:58.926559Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:08:59.998518Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:01.006854Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:01.008967Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:04.653309Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:04.658243Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:04.964967Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:05.431029Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:06.669157Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:06.669361Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:06.776459Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:06.776753Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:06.878616Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:06.878631Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:06.984490Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.014373Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.092394Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.124334Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.202917Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.233934Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.307380Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.341026Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.417771Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.448115Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.526849Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.558246Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.625522Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.669103Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.729429Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.803344Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.832616Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.925876Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:07.938697Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:08.052356Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:08.052356Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:08.176949Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:08.177818Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:08.278013Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:08.294798Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:08.404324Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:08.409328Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:10.593177Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:11.206278Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:24.824989Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:48.912930Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:49.528307Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:51.304641Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:53.595706Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:57.614160Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:58.094820Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:09:58.478304Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:00.377827Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:00.890610Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:04.693940Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:05.595158Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:07.922447Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:15.650354Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:16.001689Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:18.324499Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:19.378604Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:20.029394Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:20.569633Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:21.375752Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:22.394094Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:23.434937Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:24.900492Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:25.920520Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:32.159132Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:32.762252Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:34.819066Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:34.886609Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:40.774128Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:41.567834Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:41.858092Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:10:42.429053Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:11:05.265441Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:11:06.216424Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:11:11.072096Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:11:11.073487Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:11:11.807043Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:11:15.430899Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:21.506789Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:22.317056Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:23.130393Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:23.282970Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:23.436237Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:23.589720Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:23.738851Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:23.896983Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:24.052242Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:24.205055Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:24.374202Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:24.535655Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:19:24.691927Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:34.933461Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:23:34.933910Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:23:39.101750Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:39.514386Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:40.369747Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:40.382534Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:41.371862Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:41.666417Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:45.233010Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:47.089532Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:47.115966Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:47.410145Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:51.538955Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:55.191726Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:55.970890Z ERROR yazi_plugin::isolate::peek: RuntimeError("[string \"ouch\"]:39: attempt to call a nil value (field 'Text')\nstack traceback:\n\t[string \"ouch\"]:39: in function <[string \"ouch\"]:3>")
    at yazi-plugin\src\isolate\peek.rs:50

  2024-12-01T19:23:58.507527Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
C:\vfcompat.dll
C:\appverifUI.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T19:24:13.397930Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T19:24:33.398972Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:24:33.398996Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:24:38.210284Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-01T19:25:53.104596Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\comptia-security-sy0-701-exam-objectives-(5-0).pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-01T19:25:53.105030Z ERROR yazi_scheduler::prework::prework: Returned 0 when running preloader `pdf` with `C:\Users\LENOVO\Downloads\CompTIA+Security++(SY0-701)Study+Plan.pdf`
    at yazi-scheduler\src\prework\prework.rs:76

  2024-12-02T03:16:57.164068Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-02T03:16:57.164746Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-02T03:16:57.164748Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-02T03:16:57.167210Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-02T03:16:57.167217Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-02T03:16:57.169694Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-02T03:22:38.196066Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T03:26:48.632151Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T03:26:55.282910Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-02T03:26:55.282959Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-02T03:26:55.282961Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-02T03:26:55.284357Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-02T03:26:55.284368Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-02T03:26:55.286101Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-02T03:37:16.283171Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T03:45:19.917454Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-02T03:45:19.917503Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-02T03:45:19.917505Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-02T03:45:19.919108Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-02T03:45:19.919121Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-02T03:45:19.921651Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-02T03:45:25.279009Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T05:51:48.348045Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T05:53:27.640164Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-02T05:53:27.640192Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-02T05:53:27.640195Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-02T05:53:27.642183Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-02T05:53:27.642193Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-02T05:53:27.644460Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-02T05:53:29.489305Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T05:53:31.908797Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T05:53:33.448590Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T05:53:41.912253Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-02T07:23:07.769524Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-03T03:32:13.053784Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-03T03:32:13.054807Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-03T03:32:13.054812Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-03T03:32:13.058383Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-03T03:32:13.058396Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-03T03:32:13.061139Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-03T04:42:14.870528Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-03T04:42:14.871625Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-03T04:42:14.871645Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-03T04:42:14.874968Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-03T04:42:14.875008Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-03T04:42:14.879996Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-03T16:46:45.290608Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-03T16:46:45.293078Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-03T16:46:45.293133Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-03T16:46:45.296634Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-03T16:46:45.296684Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-03T16:46:45.306845Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-03T16:47:40.662555Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-03T16:47:40.662577Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-03T16:47:40.662578Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-03T16:47:40.664417Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-03T16:47:40.664425Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-03T16:47:40.666098Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-03T16:52:29.591127Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-03T16:54:24.926672Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-03T16:54:24.926702Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-03T16:54:24.926705Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-03T16:54:24.928928Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-03T16:54:24.928940Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-03T16:54:24.932161Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-03T17:02:32.794132Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-03T17:02:39.855295Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-03T17:04:35.787270Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\Users\LENOVO\.bash_history
C:\Users\LENOVO\.gitconfig
C:\Users\LENOVO\.lesshst
C:\Users\LENOVO\.node_repl_history
C:\Users\LENOVO\.vivaldi_reporting_data
C:\Users\LENOVO\battery-report.html
C:\Users\LENOVO\NTUSER.DAT
C:\Users\LENOVO\ntuser.dat.LOG1
C:\Users\LENOVO\ntuser.dat.LOG2
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TM.blf
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000001.regtrans-ms
C:\Users\LENOVO\NTUSER.DAT{a2332f18-cdbf-11ec-8680-002248483d79}.TMContainer00000000000000000002.regtrans-ms
C:\Users\LENOVO\ntuser.ini
C:\Users\LENOVO\y.cmd
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-03T17:04:38.480898Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\DumpStack.log.tmp
C:\hiberfil.sys
C:\pagefile.sys
C:\swapfile.sys
C:\SYSTAG.BIN
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-03T17:04:41.556483Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
C:\DumpStack.log.tmp
C:\hiberfil.sys
C:\pagefile.sys
C:\swapfile.sys
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-04T07:10:46.398057Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-04T07:10:46.399993Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-04T07:10:46.399998Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-04T07:10:46.402734Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-04T07:10:46.402758Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-04T07:10:46.407459Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-06T03:30:14.315020Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-06T03:30:14.315937Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-06T03:30:14.315943Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-06T03:30:14.319127Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-06T03:30:14.319148Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-06T03:30:14.325817Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-06T03:31:54.141286Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-06T03:31:54.141361Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-06T03:31:54.141364Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-06T03:31:54.143248Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-06T03:31:54.143262Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-06T03:31:54.146118Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-07T06:20:20.540391Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-07T06:20:20.541339Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-07T06:20:20.541343Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-07T06:20:20.544419Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-07T06:20:20.544442Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-07T06:20:20.548547Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-07T10:19:13.717315Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-07T10:19:13.717480Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-07T10:19:13.717486Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-07T10:19:13.721492Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-07T10:19:13.721520Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-07T10:19:13.727188Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-07T14:40:21.577853Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-07T14:40:21.578654Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-07T14:40:21.578659Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-07T14:40:21.581564Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-07T14:40:21.581583Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-07T14:40:21.585081Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-08T12:36:43.277534Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-08T12:36:43.278873Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-08T12:36:43.278876Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-08T12:36:43.281450Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-08T12:36:43.281461Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-08T12:36:43.283883Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-09T12:48:04.261364Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-09T12:48:04.262074Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-09T12:48:04.262077Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-09T12:48:04.264299Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-09T12:48:04.264312Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-09T12:48:04.266634Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-09T13:01:24.413639Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-09T13:01:24.413686Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-09T13:01:24.413688Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-09T13:01:24.414987Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-09T13:01:24.414995Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-09T13:01:24.416569Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-09T13:37:22.012466Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-09T13:37:22.012487Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-09T13:37:22.012489Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-09T13:37:22.013269Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-09T13:37:22.013282Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-09T13:37:22.015553Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-10T10:26:02.166230Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-10T10:26:02.167723Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-10T10:26:02.167730Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-10T10:26:02.171299Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-10T10:26:02.171315Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-10T10:26:02.173956Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T01:31:56.122872Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T01:31:56.122923Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T01:31:56.122925Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T01:31:56.123911Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T01:31:56.123920Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T01:31:56.126056Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T01:40:08.507744Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:/appverifUI.dll
C:/DumpStack.log
C:/vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T01:41:09.317594Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T01:41:09.317639Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T01:41:09.317640Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T01:41:09.319397Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T01:41:09.319404Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T01:41:09.321115Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T02:11:37.610682Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T02:11:37.610731Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T02:11:37.610732Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T02:11:37.612393Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T02:11:37.612400Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T02:11:37.613950Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T02:14:31.510947Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T02:14:31.511000Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T02:14:31.511001Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T02:14:31.512845Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T02:14:31.512853Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T02:14:31.514505Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T02:15:01.291913Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T02:15:01.291960Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T02:15:01.291961Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T02:15:01.293564Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T02:15:01.293572Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T02:15:01.295009Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:17:09.794241Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T04:17:09.794290Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T04:17:09.794292Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T04:17:09.795102Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:17:09.795113Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:17:09.797636Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:17:12.808795Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:18:20.901768Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T04:18:20.901819Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T04:18:20.901821Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T04:18:20.902785Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:18:20.902801Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:18:20.905526Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:18:23.558425Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:25:09.979331Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T04:25:09.979379Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T04:25:09.979381Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T04:25:09.980304Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:25:09.980311Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:25:09.981864Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:25:12.299604Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:37:54.631320Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:37:54.631340Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:37:54.633212Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:38:03.915827Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:38:11.391823Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:38:11.391850Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:38:11.396115Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:39:20.121476Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:39:20.121501Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:39:20.123356Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:39:25.085814Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:39:25.085848Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:39:25.087693Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:40:01.407333Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:40:01.407369Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:40:01.409845Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:41:21.567212Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:41:21.567232Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:41:21.568808Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:41:25.039167Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:41:36.082162Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:41:36.082182Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:41:36.083704Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:41:52.919826Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:41:52.919848Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:41:52.922424Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:41:54.824057Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:41:57.383182Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:42:03.381582Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:42:08.873057Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:42:08.873077Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:42:08.876016Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:42:10.817065Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:43:56.830198Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:43:56.830216Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:43:56.832100Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:43:58.202350Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:59:17.292177Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:59:17.292196Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:59:17.293922Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:59:19.548171Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:59:22.378830Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:59:36.813885Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T04:59:36.813904Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T04:59:36.815564Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T04:59:42.847913Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T04:59:44.729293Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\DumpStack.log
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T08:57:46.552742Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T08:57:46.554737Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T08:57:46.557734Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T08:57:48.841050Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T09:46:00.085510Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T09:46:00.085565Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T09:46:00.085568Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T09:46:00.087523Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T09:46:00.087532Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T09:46:00.089684Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T09:47:56.246017Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T09:47:56.246074Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T09:47:56.246077Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T09:47:56.247308Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T09:47:56.247317Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T09:47:56.249216Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T09:52:22.474067Z ERROR yazi_scheduler::prework::prework: Returned 2 when running fetcher `mime` with:
C:\appverifUI.dll
C:\DumpStack.log
C:\vfcompat.dll
    at yazi-scheduler\src\prework\prework.rs:49

  2024-12-12T09:57:01.388772Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T09:57:01.388796Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T09:57:01.388798Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T09:57:01.390192Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T09:57:01.390223Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T09:57:01.393127Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T09:58:50.121352Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T09:58:50.121398Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T09:58:50.121400Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T09:58:50.122347Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T09:58:50.122360Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T09:58:50.124572Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

  2024-12-12T10:13:17.932817Z  WARN yazi_adapter::emulator: [Adapter] No special environment variables detected
    at yazi-adapter\src\emulator.rs:73

  2024-12-12T10:13:17.932869Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM_PROGRAM: 
    at yazi-adapter\src\emulator.rs:87

  2024-12-12T10:13:17.932871Z  WARN yazi_adapter::emulator: [Adapter] Unknown TERM: 
    at yazi-adapter\src\emulator.rs:95

  2024-12-12T10:13:17.934474Z  WARN yazi_adapter::adapter: [Adapter] Could not identify XDG_SESSION_TYPE
    at yazi-adapter\src\adapter.rs:108

  2024-12-12T10:13:17.934483Z  WARN yazi_adapter::adapter: [Adapter] Falling back to chafa
    at yazi-adapter\src\adapter.rs:120

  2024-12-12T10:13:17.936255Z  WARN yazi_adapter::ueberzug: ueberzugpp spawning failed: program not found
    at yazi-adapter\src\ueberzug.rs:96

